docs/using-lagoon-the-basics/docker-compose-yml.md

```diff
@@ -94,6 +94,9 @@ This defines all the services you want to deploy. _Unfortunately,_ `docker-compo
 
 The name of the service \(`nginx`, `php`, and `mariadb` in the example above\) is used by Lagoon as the name of the Kubernetes pod \(yet another term - again, we'll be calling them services\) that is generated, plus also any additional Kubernetes objects that are created based on the defined `lagoon.type`, which could be things like services, routes, persistent storage, etc.
 
+!!! warning "Warning:"
```

Comment:
I'd also set this warning under multi-container pods - i.e. changing the lagoon.name will have the same effect